packages/mobile/src/escrow/saga.ts

```diff
@@ -158,20 +158,21 @@ async function createReclaimTransaction(paymentID: string) {
   return escrow.methods.revoke(paymentID)
 }
 
-export async function getReclaimEscrowFee(account: string, paymentID: string) {
-  // create mock transaction and get gas
+export async function getReclaimEscrowGas(account: string, paymentID: string) {
```

Comment:
You will see this pattern for all the tx types. I had to break apart the gas part so we can cache that. Now we only need to refresh the gas price and multiply it by the cached gas amount. Noticeably quicker for repeat txs.